amalgamation/Makefile

```diff
@@ -17,8 +17,8 @@ endif
 
 CFLAGS=-std=c++11 -Wno-unknown-pragmas -Wall
 ifneq ($(MIN), 1)
-	CFLAGS+= -I${OPENBLAS_ROOT}/include
-	LDFLAGS+=-L${OPENBLAS_ROOT}/lib -lopenblas
+	CFLAGS+= -I${OPENBLAS_ROOT}
```

Comment:
${OPENBLAS_ROOT}/include and ${OPENBLAS_ROOT}/lib only exist if you make install OpenBLAS to ${OPENBLAS_ROOT}. Since:
1. The instructions on building OpenBLAS for Android does not include 'make install';
2. OpenBLAS's header files and compiled lib will remain in the building directory even if you make install it;
3. OpenBLAS's header files and compiled lib will reside in system include and library paths if you install it from system wide package manager.
I think the change on this include flag and link flag to ${OPENBLAS_ROOT} is solid.